### DISCUSSION TITLE
Does this have support for open source VR headsets?

### DISCUSSION THREAD

**Discussion by Pythone** [mozilla/hubs#4437] [General Questions & Discussion] [answered]
Is this compatible with Relativty, Vis3r, and other open source VR headsets?

**Comment by brianpeiris** [comment, marked as answer]
If those headsets support WebXR, there's a chance that Hubs could work on them.

**Reply by Pythone** [reply]
thanks will check.

**Reply by Pythone** [reply]
They are not standalone

**Reply by brianpeiris** [reply]
They don't need to be standalone. Hubs works with tethered headsets like the Valve Index, HTC Vive and Windows Mixed Reality headsets too.

**Reply by Pythone** [reply]
good.
